docs/guide/protocols/absolutesolvation.rst

```diff
@@ -1,4 +1,23 @@
 Absolute Solvation Protocol
 ===========================
 
-issue #743
\ No newline at end of file
+The absolute solvation protocol calculates the free energy change 
```

Comment:
The content here is good but we might need to reformat things to make it easier for readers to digest / quickly understand what they want.

I would propose we break down the page into the following subheadings:

```rst
Overview
~~~~~~

A short overview of the protocol (what you have all the way to the figure).

Scientific Details
~~~~~~~~~~~

Here we can talk about how the Protocol is a partial annhilation scheme, the lambda schedule, softcoring, what estimatse you get out (i.e. what you can get out of the results object's methods).

No need to go into a lot of prose here, you can keep things brief and just have sub-headings that effetively act as major bullet points.

We can also include information about ensembles, cutoffs, and integrators here too.

Simulation overview
~~~~~~~~~~~~~

This section should essentially give an overview of the DAG and what each unit is doing.

For example we would want to say that each unit is doing a non-alchemical equilibration followed by an alchemical production.

We would also mention how the DAG constructs and runs both the vacuum and solvent legs concurrently.

See Also
~~~~~~

See my comment below about how we could signpost folks to different tutorials, cookbooks, etc... that are relevant to this Protocol.

References
~~~~~~~
Some relevant references that folks can look at, maybe links to pymbar/yank/perses/openmmtools/openmm/etc...
```